### DISCUSSION TITLE
ability to make a link to someones profile

### DISCUSSION THREAD

**Discussion by ghost** [stoatchat/discussions#247] [Feature Suggestions]
There should be an ability to create or copy a link which leads to your profile. E.g. https://app.revolt.chat/user/exampleusername or https://app.revolt.chat/user/exampleuuid

**Comment by Rexogamer** [comment]
If discriminators are added, this either might have to be just UUIDs or have some sort of suffix (e.g. https://app.revolt.chat/user/testuser-1010)

**Reply by ghost** [reply]
I personally think that uuids make more sense as they stay the same even if you change your username
